modules/iam-role-for-serviceaccount/variables.tf

```diff
@@ -44,3 +44,9 @@ variable "tags" {
   type        = map(string)
   default     = {}
 }
+
+variable "wildcard_namespace" {
```

Comment:
What do you think about making this variable a string that represents the condition operator?  It could default to `StringEquals` but can be overridden to any of the 5 other string operators specified in [IAM JSON policy elements: Condition operators](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_condition_operators.html)

Reply:
I hadn't considered that, good idea!!!